### DISCUSSION TITLE
biglybt/vuze webui support

### DISCUSSION THREAD

**Discussion by DJGoo** [pymedusa/Medusa#11829] [Ideas & Feature requests]
BiglyBT works with Medusa if I use the MLDonkey setting, but it doesn't have any options for setting a category or download location.

BiglyBT’s tags function the same way as qBittorrent’s categories, just more customizable.

Sonarr supports Vuze (which is just an older version of BiglyBT), including reading the seed ratio and time, and it also supports categories. So, it's possible to add similar support to Medusa, although I'm not sure how difficult that would be.

**Comment by nydave69** [comment]
I would like to see this myself..  +1
